packages/combo-box/src/vaadin-combo-box-scroller.js

```diff
@@ -311,6 +311,10 @@ export class ComboBoxScroller extends PolymerElement {
     } else {
       el.removeAttribute('theme');
     }
+
+    if (item instanceof ComboBoxPlaceholder) {
```

Comment:
Extracted this condition from the method as it kind of feels unrelated to the method. Or otherwise, the method should probably be called `__requestItemByIndexIfNeeded`. If you feel differently, I can revert this change, not a big deal.

Reply:
The change makes sense IMO. The current logic is a leftover from before the `<iron-list>` -> `Virtualizer` refactor.